### PR TITLE
functional-tester: share limiter among stresser

### PR DESCRIPTION
**Pull request by heyitsanthony**
Otherwise, adding more members stresses the cluster with more ops.

/cc @xiang90 @gyuho

### CODE REVIEW

File: tools/functional-tester/etcd-tester/stresser.go [hunk 1]
```diff
@@ -180,7 +179,9 @@ func (s *stresser) Stress() error {
 	s.conn = conn
 	s.cancel = cancel
 	s.wg = wg
-	s.rateLimiter = rate.NewLimiter(rate.Limit(s.qps), s.qps)
+	if s.rateLimiter == nil {
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/stresser.go]: probably just panic here? we do not expect this to be unset, right?

**Comment by xiang90** [comment]
lgtm

File: tools/functional-tester/etcd-tester/stresser.go (hunk 1 above)
Comment (heyitsanthony) [inline tools/functional-tester/etcd-tester/stresser.go]: ok

**Comment by gyuho** [comment]
LGTM. Thanks!

**Pull request merged**